hamocc/mo_hamocc_init.F90

```diff
@@ -203,6 +204,7 @@ subroutine hamocc_init(read_rest,rstfnm_hamocc)
        call ini_read_ndep(idm,jdm)
     end if
     call ini_read_rivin(idm,jdm,omask)
+    call read_shelfmask(idm,jdm,nbdy,depths,omask)
```

Comment:
I would prefer the name `ini_shelfmask`, since this is what the routine does (yes, it might read a file for doing this, but not necessarily) and for consistency with the other names.

Reply:
renamed to `ini_read_shelfmask`